pkg/timelock/scheduler.go

```diff
@@ -216,7 +217,7 @@ func writeOperationStore(
 	for _, record := range storeRecords {
 		op = record.Ops[0]
 
-		if int(op.Raw.BlockNumber) == earliest {
```

Comment:
type conversion for the comparison against an int here is unneeded since we now pass down a `uint64`